docs/UserGuide.md

```diff
@@ -187,6 +187,15 @@ in author-config.csv, add <a href="#standalone-configuration">Standalone Configu
 Otherwise, all the authors of the repositories will be added into the report by default.
 </h5>
 
+Column Name | Explanation
+----------- | -----------
+Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
+[Optional] Branch | The branch to analyse in the target repository
+Author's GitHub ID | GitHub ID of the target contributor in the repository
+Author's Display Name | Optional Field. The value of this field, if not empty, will be displayed in the dashboard instead of author's GitHub ID.
+[Optional] Author's Git Author Name | Detailed explanation below
+[Optional] Ignore Global List | The list of file path globs to ignore during analysis for this author on top of what is already specified in `author-config.csv`. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+
```

Comment:
is this entire change suppose to be here?

Reply:
Checked with the UserGuide currently in master branch, change appears to be there

Reply:
> Checked with the UserGuide currently in master branch, change appears to be there

As in, things that are suppose to be part of your PR, should not appear in it. 
If its due to the merging, please do a clean merge.

Reply:
I don't understand what you are referring to, this change was not done by me therefore it is not a part of my PR. This change is a part of the many other changes made by other contributors which has been merged.

Reply:
Merge has been cleaned

Reply:
Still seems to be here somehow. 
You may have accidentally combined a merge and your changes in the same commit, which results in its appearance. 

Could you please do a rebase merge to get rid of it? 

While its a small part that can be easily ignored, its not favorable to display/include changes that are irrelevant to the PR.